android/engine/src/test/java/org/smartregister/fhircore/engine/util/DateUtilsTest.kt

```diff
@@ -38,6 +39,27 @@ class DateUtilsTest {
     Assert.assertEquals("3-22-2020", finalDate)
   }
 
+  @Test
+  fun `addDays() should return same date when given initialDate and pattern`() {
+    val finalDate =
+      DateUtils.addDays("2020-03-10 01:23:00 AM", dateTimeFormat = "yyyy-MM-dd h:mm:ss a")
+    Assert.assertEquals("3-10-2020", finalDate)
+  }
+
+  @Test
+  fun `hasPastDays should return true when given initialDate and days`() {
```

Comment:
Can we re-word this to capture conditions under test and expected output?

Reply:
Is this what you have in mind?
`hasPastDays() should return true when given valid initialDate and has negative days`

Reply:
maybe something like `hasPastDays() should return true when given  date is 3 days in the past`

Reply:
Yeah, that makes sense